src/scripts/store/dimStoreItem.directive.js

```diff
@@ -91,6 +91,8 @@ function StoreItem(dimItemService, dimStoreService, ngDialog, dimLoadoutService,
     vm.clicked = function openPopup(item, e) {
       e.stopPropagation();
 
+      $rootScope.$broadcast('item-clicked', item);
```

Comment:
I'm very concerned about this - I'm trying to eliminate digests and global events, and this introduces a pretty big one.